README.md

```diff
@@ -104,70 +104,128 @@ that more automatic in the end.
 
 To test a small set of files to see whether the workflows run smoothly, run:
 ```
-python runner.py --workflow ${workflow} --json metadata/test_w_dj.json 
+python runner.py --workflow ${workflow} --json metadata/test_w_dj.json --campaign Rereco17_94X --year 2017
```

Comment:
`test_w_dj.json` does not exist anymore

Suggest giving an explicit example workflow here, e.g. `ttsemilep_sf` with the json file `test_w_dj_mu.json`

Lines 97-103 are outdated, suggest deleting them